packages/xstate-graph/test/shortestPaths.test.ts

```diff
@@ -131,7 +131,7 @@ describe('getShortestPaths', () => {
           todo: 'two'
         } as const
       ],
-      filter: (state) => state.context.todos.length < 3
+      stopWhen: (state) => state.context.todos.length >= 3
```

Comment:
this feels like a breaking change that wasn't mentioned in changesets

Reply:
I will add a changeset in a follow-up (clean-up) PR

Reply:
#4911